Services/CO.CDP.OrganisationInformation.Persistence/Organisation.cs

```diff
@@ -15,7 +15,8 @@ public class Organisation : IEntityDate
     public ICollection<OrganisationAddress> Addresses { get; set; } = [];
     public required OrganisationContactPoint ContactPoint { get; set; }
     public List<PartyRole> Roles { get; set; } = [];
-    public List<Person> Persons { get; } = [];
+    public List<Person> Persons => OrganisationPersons.Select(p => p.Person).ToList();
```

Comment:
I'm uncertain if this is going to cause our problems. I wanted us to always create new person connections via the OrganisationPersons link rather than adding to the Persons collection. OrganisationPersons has the additional Scopes field.